kafka-ui-react-app/src/components/Topics/Topic/SendMessage/validateMessage.ts

```diff
@@ -3,7 +3,7 @@ import Ajv from 'ajv/dist/2020';
 import { upperFirst } from 'lodash';
 
 const validateBySchema = (
-  value: string,
+  value: string | undefined,
```

Comment:
can't we just write 
`value?: string ,`

Reply:
fixed